src/Language/Writer/NullValueWriter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Digia\GraphQL\Language\Writer;
+
+use Digia\GraphQL\Language\AST\Node\NodeInterface;
+use Digia\GraphQL\Language\AST\Node\NullValueNode;
+
+class NullValueWriter extends AbstractWriter
+{
+    /**
+     * @param NodeInterface|NullValueNode $node
+     * @inheritdoc
+     */
+    public function write(NodeInterface $node): string
```

Comment:
If we return null here why we need the $node?

Reply:
It's the interface, it's required for all the other writers.